### DISCUSSION TITLE
syntax error on line 65 for password - expecting ']'

### DISCUSSION THREAD

**Discussion by Bob-839** [YOURLS/YOURLS#3458] [General]
I received this error: Parse error: syntax error, unexpected '$yourls_user_passwords' (T_VARIABLE), expecting ']' in /home/customer/www/pk9t-bob.com/public_html/user/config.php on line 65

The code is intact still, I did not alter it, except to change user and password. Any help would be appreciated thanks.

**Comment by ozh** [comment]
You didn't close the list of passwords with a `]`, apparently. See the provided sample -- https://github.com/YOURLS/YOURLS/blob/master/user/config-sample.php#L65-L69
